### ISSUE TITLE
CsvWriter doesn't escape quoted value

### ISSUE THREAD

**Issue by aleskobersky** [uniVocity/univocity-parsers#34]
## Problem
The CsvWriter doesn't quote a column which contains a quote character `"`. 
A single quote as value should be imho coded as `""""`

## Details
Please see the code snippet below. There are 3 columns on the input:

| Col1 | Col 2 | Col 3 |
| --- | --- | --- |
| Quote | " | Value with quote " |

The output line looks like:

`Quote,",Value with quote "`

Which is then parsed to:

| Col1 | Col 2 |
| --- | --- |
| Quote | ,Value with quote |

## Code snippet
```java
		Object[] data = new Object[] {"Quote", "\"", "Value with quote\""};
		
		File file = File.createTempFile("Test", "csv");
		CsvWriter writer = new CsvWriter(new FileWriter(file), new CsvWriterSettings());
		writer.writeRow(data);
		writer.close();
		
		List<String> lines = Files.readAllLines(file.toPath(), Charset.defaultCharset());
		for(String line : lines) {
			System.out.println("Output line:" + line);
		}
		
		System.out.println("Parsed data:");
		CsvParser reader = new CsvParser(new CsvParserSettings());
		List<String[]> parsedData = reader.parseAll(new FileReader(file));
		for(String[] line : parsedData) {
			for (int i = 0; i < line.length; i++) {
				System.out.println("Col" + i + ":" + line[i]);
			}
		}
```
_The snippet is quick&dirty but serves the purpose ;-)_

**Comment by jbax** [comment]
Good timing! This was fixed yesterday in commit: https://github.com/uniVocity/univocity-parsers/commit/8c195d96ef3e13554798ce68d5ba4301a97bdc37

I'll build a 2.0.0 SNAPSHOT soon and backport the fix for this and a few other issues to a new build version 1.5.6

**Comment by jbax** [comment]
Just released version 1.5.6 with fixes for this and other issues.

Direct link for the jar [here](http://oss.sonatype.org/content/repositories/releases/com/univocity/univocity-parsers/1.5.6/univocity-parsers-1.5.6.jar) (the central repository might need a few hours to make the artifact available through maven).
